frontend/pages/register/passport/passport.module.css

```diff
@@ -0,0 +1,62 @@
+.button {
+  align-self: center;
+  background-color: var(--darkBlue);
+  color: white;
+  font-size: 19px;
+  height: var(--spacing8);
+  margin-bottom: 0.5em;
+  width: var(--spacing10);
+}
+
+.container {
```

Comment:
I am not sure about the .button class naming but could we switch from css flex to css grid? its far more declarative and descriptive for what it is actually doing https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
Yup, I can refactor that out, especially since I've seen the mobile mocks and have a clearer idea of what this should look like on all screens.